### DISCUSSION TITLE
homepage style

### DISCUSSION THREAD

**Discussion by u-an-i** [albar965/littlenavmap#672] [General]
enter

`document.head.innerHTML += "<link href='https://uani.de/support/look.css' rel='stylesheet' type='text/css'>"`

into the console of the web debugger of your browser on https://albar965.github.io and press enter.

What do you think?

![image](https://user-images.githubusercontent.com/84718885/120972169-c0816400-c76d-11eb-80ee-2186ba02e93e.png)

**Comment by albar965** [comment]
I get a tiny frame in the middle of the window. But gimme some time. I'd like to get this release out of the door first.

Feel free to experiment with this repo: https://github.com/albar965/albar965.github.io

Serve locally on Linux with 

``` bash
#!/bin/bash
(
cd $APROJECTS/albar965.github.io
RUBYOPT='-W0' bundle exec jekyll serve -H sol --drafts --future -P 4001
)

```

**Reply by albar965** [reply]
Somehow the view is too small. Latest Firefox, Linux.
![grafik](https://user-images.githubusercontent.com/15254956/121544283-e460d600-ca09-11eb-8ee3-f2e656a2a6b1.png)

**Reply by u-an-i** [reply]
i was inspired by https://openlayers.org which reminded me the background image I like to see but there the map at the top is much too small for my taste.

I set it full-height now almost. I allowed myself to add a hover effect which hides the text when the mouse is along a page border.

**Reply by albar965** [reply]
I would not mind a better look. Looks kinda old fashioned what I have there right now.

**Reply by u-an-i** [reply]
I just saw my Ff caches. If you like to see the update, clear the cache of Ff. Win Ff in contrast to Win Chrome jitters the animation.
